splitio/version.go

```diff
@@ -2,4 +2,4 @@
 package splitio
 
 // Version is the version of this Agent
-const Version = "5.0.2"
+const Version = "5.0.3-rc1"
```

Comment:
Given the new features, would it fit as a `5.1.0-rc1` update?
Also: don't forget the `changes` file